pygeoapi/api/__init__.py

```diff
@@ -829,8 +831,10 @@ def landing_page(api: API,
                                     'type', 'collection'):
             fcm['collection'] = True
 
-        content = render_j2_template(api.tpl_config, 'landing_page.html',
-                                     fcm, request.locale)
+        content = render_j2_template(
+            api.config, api.config['server']['templates'], 'landing_page.html',
```

Comment:
Like above, want to use `api.tpl_config`
```suggestion
            api.tpl_config, api.config['server']['templates'], 'landing_page.html',
```